ReplaySample/src/main/assets/html/hidden.html

```diff
@@ -0,0 +1,147 @@
+<!DOCTYPE html>
```

Comment:
All these HTML files aren't used. Please remove the entire assets folder - nothing within it is used.